react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -179,12 +189,13 @@ public void onReactContextInitialized(ReactContext context) {
     }
 
     public void attachToContainer(ViewGroup viewGroup, OnMediaLibraryButtonListener onMediaLibraryButtonListener,
-                                  OnReattachQueryListener onReattachQueryListener) {
+                                  OnReattachQueryListener onReattachQueryListener, OnEditorMountListener onEditorMountListener) {
```

Comment:
Let's wrap this to the next line so it doesn't exceed the 120char mark. The linter will fail otherwise on CI.

Reply:
Fix in: aa3435b4d384a9ca61a7dc43aa5ab8449936794a